docs/types/at-a-glance.md

```diff
@@ -35,4 +35,8 @@ The Pony type system offers a lot of guarantees, even more than other statically
 
 Some of those will make sense right now. Some of them may not mean much to you yet (like capabilities-security and causal messaging), but we'll get to those concepts later on.
 
-__If I use Pony's FFI to call code written in another language, does Pony magically make the same guarantees for the code I call?__ Sadly, no. Pony's type system guarantees only apply to code written in Pony. Code written in other languages gets only the guarantees provided by that language.
+__If I use Pony's FFI to call code written in another language, does Pony magically make the same guarantees for the code I call?__ Sadly, no. Pony's type system can only guarantee code written in Pony. Code written in other languages gets only the guarantees provided by that language.
```

Comment:
Personal writing style comment here so feel free to ignore. I do not like the use of "magically" in this context as the question is asked from the perspective of the reader/learner so it seems dismissive to suggest they expect some "magic" -- although I certainly know I have expected magic before. Can you change this to be more open-ended and without the magic such as:
> __If I use Pony's FFI to call code written in another language, can Pony make the same guarantees for the code I call?__

Reply:
It's not my writing style since I didn't write that section :-) ... but I'm happy to make that change if you think it's an improvement.

Reply:
I agree with Ryan, I'd drop the "magic" term, the original sentence still works without it.